pkg/controller/disruption/disruption.go

```diff
@@ -326,12 +326,29 @@ func (dc *DisruptionController) implementsScale(gv schema.GroupVersion, kind str
 	if err != nil {
```

Comment:
rather than passing in the `kind` and trying to locate the scale subresource, pass in the resource name from the GroupResource we used to make the scale API request, and look for `gr.Resource + "/scale"` in discovery

Reply:
Updated. Now relies on the RESTMapper getting the right `Kind` -> `Resource` translation and then asserts the scale subresource data format matches the expected scale formats.

Reply:
much cleaner / simpler, thanks